Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -407,6 +407,24 @@ class ExportBrowserDataSetting: HiddenSetting {
     }
 }
 
+class EnableActivtyStreamSetting: HiddenSetting {
+    let profile: Profile
+
+    override init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+        super.init(settings: settings)
+    }
+
+    override var title: NSAttributedString? {
+        // Not localized for now.
+        return NSAttributedString(string: "Enable the New Tab Experience", attributes: [NSForegroundColorAttributeName: UIConstants.TableViewRowTextColor])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        FeatureSwitches.activityStream.enrollMember(profile.prefs)
```

Comment:
I dont think allowing disabling of AS will be a good idea. After this feature is released to a general audience we'll be stripping out the old Topsites panel and its not something we want to continue to support. If someone disables it and then in a later update has it reenabled once we strip out the old panel they'll be angry. If someone doesnt like AS we should be recommending them to change their default first panel rather than go back to the old panel.